mesos/scheduler.go

```diff
@@ -338,7 +338,11 @@ func (s *Scheduler) ScheduleTask(request eremetic.Request) (string, error) {
 		"ports":             request.Ports,
 	}).Debug("Adding task to queue")
 
-	task, err := eremetic.NewTask(request, fmt.Sprintf("Eremetic task %s", nextID(s)))
+	if request.Name == "" {
```

Comment:
I don't remember why we set the task name here and not in NewTask() but now it strikes me as kinda odd. I don't think it's a big deal but would it makes sense to move it?

Reply:
I think its due to `nextID()` logic

Reply:
ah, yes, I see that now. Untangle that might be a bigger thing so lets not bundle that with this change.

Needs to be added to the swagger but otherwise LGTM!

Reply:
oh yeah, I missed that! going to add it. Thanks!